powerauth-push-model/src/main/java/io/getlime/push/model/response/ListOfUsersPagedResponse.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2016 Lime - HighTech Solutions s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getlime.push.model.response;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.getlime.push.model.customization.CustomPagedListOfUsersDeserializer;
+import io.getlime.push.model.base.PagedResponse;
+
+/**
+ * Paged response class used in retrieving list of users from campaign
+ *
+ * @author Martin Tupy, martin.tupy.work@gmail.com
+ */
+
+@JsonDeserialize(using = CustomPagedListOfUsersDeserializer.class)
+public class ListOfUsersPagedResponse<ListOfUsersFromCampaignResponse> extends PagedResponse<ListOfUsersFromCampaignResponse> {
```

Comment:
Concrete paged response for list of users, with custom deserializer. Default deserializer couldn't be used because TypeRefernce had deep generics and mapper couldn't recognize JSon nodes.

Reply:
We do not need to wrap `PagedResponse` in `ObjectResponse`...

Reply:
Problem associated with deserialization, see below. At this point this is our returning object from server, so it has to be ObjectResponse.